code/game/machinery/computer/communications.dm

```diff
@@ -159,8 +159,10 @@
 					if(SSshuttle.emergency.mode != SHUTTLE_RECALL && SSshuttle.emergency.mode != SHUTTLE_IDLE)
 						to_chat(usr, "<span class='alert'>It's a bit late to buy a new shuttle, don't you think?</span>")
 						return
-					if(SSshuttle.shuttle_purchased)
+					if(SSshuttle.shuttle_purchased == SHUTTLEPURCHASE_PURCHASED)
 						to_chat(usr, "<span class='alert'>A replacement shuttle has already been purchased.</span>")
+					else if(SSshuttle.shuttle_purchased == SHUTTLEPURCHASE_FORCED)
+						to_chat(usr, "<span class='alert'>Due to unforseen consequences, shuttle purchasing is no longer available.</span>")
```

Comment:
```suggestion
						to_chat(usr, "<span class='alert'>Due to unforseen circumstances, shuttle purchasing is no longer available.</span>")
```

Reply:
Unforeseen consequences is fine. *Unforseen* however...